certinfo.go

```diff
@@ -98,21 +100,31 @@ func printName(names []pkix.AttributeTypeAndValue, buf *bytes.Buffer) []string {
 			switch oid[3] {
 			case 3:
 				values = append(values, fmt.Sprintf("CN=%s", name.Value))
+			case 5:
+				values = append(values, fmt.Sprintf("SERIALNUMBER=%s", name.Value))
 			case 6:
 				values = append(values, fmt.Sprintf("C=%s", name.Value))
 			case 7:
 				values = append(values, fmt.Sprintf("L=%s", name.Value))
 			case 8:
 				values = append(values, fmt.Sprintf("ST=%s", name.Value))
+			case 9:
+				values = append(values, fmt.Sprintf("STREET=%s", name.Value))
 			case 10:
 				values = append(values, fmt.Sprintf("O=%s", name.Value))
 			case 11:
 				values = append(values, fmt.Sprintf("OU=%s", name.Value))
+			case 17:
+				values = append(values, fmt.Sprintf("POSTALCODE=%s", name.Value))
 			default:
 				values = append(values, fmt.Sprintf("UnknownOID=%s", name.Type.String()))
 			}
 		} else if oid.Equal(oidEmailAddress) {
-			values = append(values, fmt.Sprintf("emailAddress=%s", name.Value))
+			values = append(values, fmt.Sprintf("MAIL=%s", name.Value))
```

Comment:
Where did you find `MAIL` as short for emailAddress?

Reply:
I saw a page where they named MAIL, but then I looked for the oid and it was emailAddress, so I reverted the change.